### PR TITLE
dashboard fixes,return requests-get api

### PR DESCRIPTION
**Pull request by KhushiSarkari**

### CODE REVIEW

File: dm-backend/Models/ReturnRequest.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+using System.Data;
+using System.Data.Common;
+using System.Threading.Tasks;
+using MySql.Data.MySqlClient;
+using dm_backend.Models;
+using static dm_backend.Utilities.Readers;
+
+namespace dm_backend.Models
+{
+    public class ReturnRequestModel
```
Comment (kaustubh-ex2) [inline dm-backend/Models/ReturnRequest.cs:12]: Can this be inherited from the Request model for reuse ?

File: dm-frontend/dashboard.html [hunk 2]
```diff
@@ -36,7 +36,7 @@
                         </div>
 
 
-                        <!--<div class="material-icons mdl-badge mdl-badge--overlap" data-badge="4">notifications</div>-->
+                        <div class="material-icons mdl-badge mdl-badge--overlap" data-badge="4">notifications</div>
```
Comment (kaustubh-ex2) [inline dm-frontend/dashboard.html:39]: Is this change required now that this feature is to be implemented ?

**Review by kaustubh-ex2 — CHANGES_REQUESTED** [review]

File: dm-frontend/dashboard.html (hunk 2 above)
Comment (KhushiSarkari) [reply dm-frontend/dashboard.html:39]: Yes, well it was excluded before so I'm including it for the same reason.

File: dm-backend/Models/ReturnRequest.cs (hunk 1 above)
Comment (KhushiSarkari) [reply dm-backend/Models/ReturnRequest.cs:12]: Yes but what about modelling?
